File: 0169-majority-element/0169-majority-element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        element=None
        count=0
        for i in range(0,len(nums)):
            if count==0:
                element=nums[i]
                count+=1
            elif element==nums[i]:
                count+=1
            elif element!=nums[i]:
                count-=1
        return element

File: 0169-majority-element/0169-majority-element.java
class Solution {
    public int majorityElement(int[] nums) {
      int e=0,c=0;
      for(int i=0;i<nums.length;i++){
        if(c==0){
            e=nums[i];
            c++;
        }
        else if(nums[i]==e){
            c++;
        }
        else{
            c--;
        }
      }
      return e;
    }
}